src/panes/histogram-pane/run-query.ts

```diff
@@ -59,15 +59,17 @@ export async function runHistogramQuery(api: ZuiApi) {
   }
 
   async function getNullTimeCount() {
-    const query = `${baseQuery} | ${timeField} == null | count()`
+    const query = `${baseQuery}
+    | ${timeField} == null | count()`
```

Comment:
I'd add a comment so no one unwittingly undoes this. And I think an escaped newline improves readability.
```suggestion
    // Newline after baseQuery in case it ends with a comment.
    const query = `${baseQuery}\n | ${timeField} == null | count()`
```